packages/core/src/html/MdAttributeRenderer.ts

```diff
@@ -75,10 +75,23 @@ export class MdAttributeRenderer {
     return hasAttribute;
   }
 
-  processPopoverAttributes(node: MbNode) {
-    if (!this.hasSlotOverridingAttribute(node, 'header')) {
-      this.processAttributeWithoutOverride(node, 'header', true);
+  /**
+   * Checks if there is a pre-existing slot for the attribute.
+   * If there is a slot, it deletes the attribute and logs a warning.
+   * If there is no slot, it processes the markdown attribute.
+   * @param node Element to process
+   * @param attribute Attribute name to process
+   * @param isInline Whether to process the attribute with only inline markdown-it rules
+   * @param slotName Name attribute of the <slot> element to insert, which defaults to the attribute name
+   */
+  processSlotAttribute(node: MbNode, attribute: string, isInline: boolean, slotName = attribute) {
+    if (!this.hasSlotOverridingAttribute(node, attribute, slotName)) {
+      this.processAttributeWithoutOverride(node, attribute, isInline, slotName);
     }
+  }
+
+  processPopoverAttributes(node: MbNode) {
+    this.processSlotAttribute(node, 'header', true);
 
     // Warn if there is a content slot overriding the attributes 'content' or 'src'
     const hasSlotAndContentAttribute = this.hasSlotOverridingAttribute(node, 'content', 'content');
```

Comment:
This bit was left unchanged because the logic for the `content` attribute is slightly different. According to the [docs](https://markbind.org/userGuide/components/popups.html#popovers), the `src`and `content` attributes and the `content` slot can be used for popovers. If all three are present then MarkBind will prioritise them in the following order:
1. `content` slot
2. `content` attribute
3. `src` attribute

So the logic to process this attribute (and especially the logger warning) is different than the other attributes. Specifically, it needs to check for the case that the `src` and `content` attributes are both used, and show a warning message if that is the case. The `processSlotAttribute` method does not account for this case, so this chunk of code is still here.